server/webhook_notifier.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"bytes"
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/applikatoni/applikatoni/deploy"
+	"github.com/applikatoni/applikatoni/models"
+)
+
+type Application struct {
+	Name        string `json:"application_name"`
+	GitHubOwner string `json:"github_owner"`
+	GitHubRepo  string `json:"github_repo"`
+}
+
+type Deployment struct {
+	Id             int                    `json:"deployment_id"`
+	CommitSha      string                 `json:"commit_sha"`
+	Branch         string                 `json:"branch"`
+	State          models.DeploymentState `json:"state"`
+	Comment        string                 `json:"comment"`
+	CreatedAt      time.Time              `json:"created_at"`
+	URL            string                 `json:"deployment_url"`
+	DeployerID     int                    `json:"deployer_id"`
+	DeployerName   string                 `json:"deployer_name"`
+	DeployerAvatar string                 `json:"deployer_avatar"`
+}
+
+type Target struct {
+	Name            string                   `json:"target_name"`
+	DeploymentUser  string                   `json:"deployment_user"`
+	DeployUsernames []string                 `json:"deploy_usernames"`
+	Hosts           []*models.Host           `json:"hosts"`
+	Roles           []*models.Role           `json:"roles"`
+	AvailableStages []models.DeploymentStage `json:"available_stages"`
+	DefaultStages   []models.DeploymentStage `json:"default_stages"`
+}
+
+type WebHookMsg struct {
```

Comment:
Could we rename this to `WebhookMsg` so that it matches the `target.Webhooks` definition?